community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/LabelScanValueIterator.java

```diff
@@ -143,4 +135,25 @@ private boolean nextFromCurrent()
         bits &= bits - 1;
         return next( baseNodeId + delta );
     }
+
+    @Override
+    public void close()
+    {
+        if ( !closed )
+        {
+            try
+            {
+                cursor.close();
+            }
+            catch ( IOException e )
+            {
+                throw new UncheckedIOException( e );
```

Comment:
Why unchecked IO exception? Wasn't it a normal one before?

Reply:
Yes it was. But the `Resource` interface does not allow `close` to throw.